### PR TITLE
Feature/moveit2 tutorials

### PR DESCRIPTION
**Pull request by hello-chintan**
Brings in the following changes from the hybrid planning branch:
- MoveIt 2 tutorial with RViz and MoveGroup C++ API
- Minor edits to the moveit config files

### CODE REVIEW

File: stretch_moveit_config/demo.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+2. Use the [Noetic installation instructions](https://github.com/hello-robot/stretch_ros/blob/dev/noetic/install_noetic.md) to create the second Ubuntu 20.04 partition, and create a ROS2 Galactic workspace with MoveIt 2 installed.
+
+3. Boot into the new Ubuntu 20.04 partition, and edit the last few lines of the ~/.bashrc file to comment out sourcing of Noetic workspaces. When both versions of ROS are sourced, package conflicts arise. The result should look like:
+
+```
+#source /opt/ros/noetic/setup.bash
+source /opt/ros/galactic/setup.bash
+#source ~/catkin_ws/devel/setup.bash
+source ~/ament_ws/install/setup.bash
+```
```
Comment (hello-binit) [inline stretch_moveit_config/demo.md]: Let's replace these steps to instead point to the [robot upgrade guide](https://docs.hello-robot.com/0.2/stretch-install/docs/robot_install/).

File: stretch_moveit_config/demo.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+6. Download additional dependencies and build the workspace using these commands. This should install MoveIt 2 and everything else you need to run the tutorials!
+
+```
+cd ~/ament_ws
+rosdep install --from-paths src --ignore-src -r --rosdistro galactic -y
+cd ~/ament_ws
+colcon build
+source ~/ament_ws/install/setup.bash
+```
```
Comment (hello-binit) [inline stretch_moveit_config/demo.md]: A robot upgrade will automatically handle these steps. If the robot has an outdated 20.04, let's recommend following the [Adding a ROS Workspace](https://docs.hello-robot.com/0.2/stretch-install/docs/ros_workspace/#ubuntu-2004) guide to create a new ament_ws.

**Review by hello-binit — CHANGES_REQUESTED** [review]
Thanks @hello-chintan, let's update the installation parts of this tutorial to point to the new Stretch Install guides.

**Comment by hello-chintan** [comment]
Done! Please let me know if anything seems amiss with the updated instructions.

**Review by hello-binit — APPROVED** [review]
Thanks! Looks good!

**Pull request merged**